net-misc/youtube-viewer/youtube-viewer-3.1.9.ebuild

```diff
@@ -7,7 +7,7 @@ EAPI=5
 inherit gnome2-utils eutils perl-module vcs-snapshot
 
 DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
-HOMEPAGE="https://trizen.googlecode.com"
+HOMEPAGE="http://trizenx.blogspot.nl/2012/03/gtk-youtube-viewer.html"
```

Comment:
Use https:// whenever possible (and it seems possible here ;-)). However, it's really trivial so we'll just change that while merging it.

Curiosity question: where did you get that homepage? I see `blogspot.ro` on GitHub. Did you get hit by some location-based redirect here? ;-)